xt_training/runner.py

```diff
@@ -165,10 +165,31 @@ def __call__(self, loader, mode=None, return_preds=False):
                     if isinstance(y, torch.Tensor):
                         y = y.to(device)
                     elif isinstance(y, Iterable):
-                        y = [y_i.to(device) for y_i in y]
```

Comment:
I'm pretty sure this will error for the data you are expecting. If y_i is a dict, then it won't have a `to` method

Reply:
I think a better practice for object detection would be to put the `to` calls inside the `forward` method.

Reply:
Thanks for the suggestion Tim. I wrote an ODInterface based on the SKInterface, and moved the `to` calls inside the `forward` method.